### PR TITLE
Fix the flashes that occur when verifying an email address.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
- Only show info once the account has been verified.
- Unify the styles for the "has this address already been registered" error with standard error dialog.
- Remove all the dead code that was left over from issue #1000 merge.

issue #1734
issue #1497
issue #218

### CODE REVIEW

File: resources/static/css/style.css [hunk 1]
```diff
@@ -515,30 +515,10 @@ button.delete:active {
...
 
-#signUpForm .red {
-  color: red;
-}
-
 #signUpForm .submit {
   height: 28px;
 }
 
-#signUpForm .error {
```
Comment (shane-tomlinson) [inline resources/static/css/style.css:526]: These styles were for the old style error messages.  The old style messages have been replaced with the newer unified error message style.

File: resources/static/pages/verify_secondary_address.js [hunk 2]
```diff
@@ -55,38 +49,37 @@ BrowserID.verifySecondaryAddress = (function() {
 
   function submit(oncomplete) {
     var pass = dom.getInner("#password") || undefined,
-        vpass = dom.getInner("#vpassword") || undefined,
```
Comment (shane-tomlinson) [inline resources/static/pages/verify_secondary_address.js:58]: This was transition code for issue #1000.  No longer needed.

File: resources/static/pages/verify_secondary_address.js [hunk 3]
```diff
@@ -103,36 +96,30 @@ BrowserID.verifySecondaryAddress = (function() {
...
     user.tokenInfo(token, function(info) {
       if (info) {
         redirectTo = info.returnTo;
         email = info.email;
         showRegistrationInfo(info);
 
-        needsPassword = info.needs_password;
         mustAuth = info.must_auth;
-
-        if (needsPassword) {
```
Comment (shane-tomlinson) [inline resources/static/pages/verify_secondary_address.js:115]: Transition code, no longer needed.

File: resources/static/css/style.css (hunk 1 above)
Comment (lloyd) [inline resources/static/css/style.css:526]: thanks for the pre-emptive explanation

File: resources/static/pages/verify_secondary_address.js (hunk 2 above)
Comment (lloyd) [inline resources/static/pages/verify_secondary_address.js:58]: ditto

**Comment by lloyd** [comment]
PRs which address lotsa things at once are hard to review well.  the actual code for the flash here is small, and I'm comfortable with it all.  doing some quick testing, running unit tests, and if that all works, merging.

**Comment by lloyd** [comment]
all manual testing looks great, no flash seen, no affect to permutations of new user / email addition in different browser / same browser.

One blocker found, frontend tests fail: pages/verify_secondary_address: valid token, no password necessary - verify user and show site info (1, 4, 5)

**Comment by lloyd** [comment]
travis agrees that this is a blocker: http://travis-ci.org/#!/mozilla/browserid/jobs/1627308

**Comment by lloyd** [comment]
r+ - better: http://travis-ci.org/#!/mozilla/browserid/builds/1628161

**Pull request merged**